### DISCUSSION TITLE
farm with nft plot

### DISCUSSION THREAD

**Discussion by jacklingbn** [Chia-Network/chia-blockchain#7593] [Support]
how do i know i won a block with nft plot?

**Comment by rmica** [comment]
Whenever feasible, I suggest to split farmers or use one farmer with split harvesters. Two boxes, one with OG and one with NFT. So the box will tell you the type of proof. In my opinion chia-blockchain still benefits from third party monitoring tools that can better adapt to specific farming architectures. I don't believe that monitoring has to be a chia-blockchain's task.

This is useful to separate the NFT proofs by the OG "true" proofs. As of now, I've understood that the pool will say you when you win a block with a NFT plot.
